### ISSUE TITLE
Curve3D endlessly prints "The axis Vector3 must be normalized" into output window

### ISSUE THREAD

**Issue by nenad-jalsovec** [godotengine/godot#83509]
### Godot version

v4.1.stable.official [970459615]

### System information

Godot v4.1.stable - Windows 10.0.19045 - Vulkan (Forward+) - dedicated NVIDIA GeForce GTX 1080 (NVIDIA; 31.0.15.3734) - Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (12 Threads)

### Issue description

The error "The axis Vector3 must be normalized" is continually printed into output window in editor when trying to do anything with the curve, like moving any point. However, it only happens for an open curve that satisfies all of the following:

- all curve points are coplanar (for example a curve whose all points have Y=0)
- has its first and last point at exactly the same position and their tangents are collinear but opposite facing.

Herse an example of a resource file that defines such curve:
~~~
[gd_resource type="Curve3D" format=3 uid="uid://nwf4scmc3p3s"]

[resource]
_data = {
"points": PackedVector3Array(-0.63, 0, 0, 0.63, 0, 0, 0.564, 0.654, 0, 0, 0, 0, 0, 0, 0, 2.553, -0.074, 0, 0, 0, 0, 0, 0, 0, 0.859, -0.848, 0, -0.635, 0, 0, 0, 0, 0, 0.564, 0.654, 0),
"tilts": PackedFloat32Array(0, 0, 0, 0)
}
point_count = 4
~~~ 

### Steps to reproduce

Make a Path3D node, with a curve that matches the described criteria (open, all points coplanar, first and last points coincide, tangents collinear but opposite facing) or load the above provided resource. Try to change the position of any point in the inspector, without breaking the mentioned criteria.

### Minimal reproduction project

N/A

**Comment by AThousandShips** [comment]
Can you please try with a recent version, like 4.1.2, your version is no longer supported and you need to make sure the bug is available on a current version

**Comment by nenad-jalsovec** [comment]
@AThousandShips Looks like the problem is fixed in 4.1.2.

**Comment by AThousandShips** [comment]
Then this can be closed, thank you for reporting
